content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -37,8 +37,10 @@ Use frontend data from RUM, as well as backend, infrastructure, and log informat
 -   You have set up [APM tracing][1] on the services targeted by your RUM applications.
 -   Your services use an HTTP server.
 -   Your HTTP servers are using [a library that supports distributed tracing](#supported-libraries).
--   You have XMLHttpRequest (XHR) or Fetch resources on the RUM Explorer to your `allowedTracingOrigins`.
--   You have a corresponding trace for requests to `allowedTracingOrigins`.
+-   You have the following set up based on your SDK:
+    - With the **Browser SDK**, you have the XMLHttpRequest (XHR) or Fetch resources on the RUM Explorer to your `allowedTracingOrigins`.
```

Comment:
```suggestion
    - With the **Browser SDK**, you have added the XMLHttpRequest (XHR) or Fetch resources on the RUM Explorer to your `allowedTracingOrigins`.
```
right? seems weird without it. But I'm questioning myself

Reply:
@kayayarai Yes, originally this word wasn't there, but it felt like something was missing to me too. Will "add" this. Thank you!